src/components/steps/StepInstitutionType.tsx

```diff
@@ -0,0 +1,96 @@
+import { useTheme } from '@mui/material';
+import { FormControl, FormControlLabel, Grid, Radio, RadioGroup, Typography } from '@mui/material';
+import { Box } from '@mui/system';
+import React from 'react';
+import { useTranslation, Trans } from 'react-i18next';
+import { OrganizationType, StepperStepComponentProps } from '../../../types';
+import { OnboardingStepActions } from '../OnboardingStepActions';
+
+type Props = StepperStepComponentProps & {
+  organizationType: OrganizationType;
+};
+
+const organizationTypeValues = [
```

Comment:
Define the correct type, using OrganizationType for the value field, in this way you will be able to use directly this value without to cast.